app/models/monitoring_review.rb

```diff
@@ -0,0 +1,33 @@
+require "api_delegator"
+
+class MonitoringReview
+  attr_accessor :access_token, :id
+
+  def initialize(id, access_token)
+    @id = id
+    @access_token = access_token
+    @errors = []
+  end
+
+  def start_date
+    get_api_data_field :attributes, :reviewStartDate
+  end
+
+  def valid?
+    review_data.error.nil?
+  end
+
+  private
+
+  def api
+    ApiDelegator.use("monitoring", "review", id: id, access_token: access_token)
+  end
+
+  def get_api_data_field(*path)
+    review_data.data.dig(*path)
```

Comment:
[question] I didn't really follow the logic of adding errors per field lookup, rather than for the actual API response not being valid, so I simplified this method.  Please let me know if I'm missing something important about the field by field error adding approach!

Reply:
The field-by-field was really needed when displaying the timeline that had an "invalid for this user" report to be able to still show that report but not link out to it or display the topics and creators.

Since errors are explicitly out of scope for this PR I think this is fine for now and will likely need to change later.

Reply:
Oooo I forgot about the timeline aspect. I will add a comment referencing this in the error issue.